.github/workflows/publish.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Python distributions to PyPI and TestPyPI
+on: 
+  push: # Runs your workflow when you push a commit or tag to main or release branches.
+      branches:
+        - main
+        - 'releases/**'
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to PyPI and TestPyPI
```

Comment:
We are not publishing to TestPyPI now, hence we modify the name to reflect that.